com.github.benpocalypse.Huely.yml

```diff
@@ -0,0 +1,23 @@
+app-id: com.github.benpocalypse.Huely
+
+runtime: io.elementary.Platform
+runtime-version: '6.1'
+sdk: io.elementary.Sdk
+
+command: com.github.benpocalypse.Huely
+
+finish-args:
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+
+  # Needed to read prefer-color-scheme with Granite.Settings
+  - '--system-talk-name=org.freedesktop.Accounts'
+
+modules:
+  - name: Huely
+    buildsystem: meson
+    sources:
+      - type: dir
```

Comment:
this will not work either since it can't find the sources. It should be a git repository commit or a tarball (prefered)